.github/workflows/build-v83.yml

```
@@ -0,0 +1,125 @@
+      # Login against a Docker registry except on PR
+      # https://github.com/docker/login-action
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # Extract the current version of PHP in Dockerfile
+      - name: Extract Current Version
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2001:style:4:20: See if you can use ${variable//search/replace} instead

(shellcheck)